src/dispatch/document/scheduled.py

```diff
@@ -48,3 +56,64 @@ def sync_document_terms(db_session=None):
         except Exception as e:
             # even if one document fails we don't want them to all fail
             log.exception(e)
+
+
+def create_reminder(db_session, owner_email, documents, contact_fullname, contact_weblink):
+    """Contains the logic for document evergreen reminders."""
+    # send email
+    plugin = plugin_service.get_active(db_session=db_session, plugin_type="email")
+    if not plugin:
+        log.warning("Document reminder not sent, no email plugin enabled.")
+        return
+
+    message_template = DOCUMENT_EVERGREEN_REMINDER
+
+    items = []
+    for doc in documents:
+        items.append(
+            {
+                "name": doc.name,
+                "description": doc.description,
+                "weblink": doc.weblink,
+            }
+        )
+    notification_type = "document-evergreen-reminder"
+    name = subject = "Document Evergreen Reminder"
+    plugin.instance.send(
+        owner_email,
+        message_template,
+        notification_type,
+        name=name,
+        subject=subject,
+        contact_fullname=contact_fullname,
+        contact_weblink=contact_weblink,
+        items=items,  # plugin expect dicts
+    )
+
+    for doc in documents:
+        doc.evergreen_last_reminder_at = datetime.utcnow()
+        db_session.add(doc)
+
+    db_session.commit()
+
+
+def group_documents_by_owner(documents):
+    """Groups documents by owner."""
+    grouped = defaultdict(lambda: [])
+    for doc in documents:
+        grouped[doc.evergreen_owner].append(doc)
+    return grouped
+
+
+@scheduler.add(every(1).day.at("18:00"), name="document-evergreen-reminder")
+@background_task
+def create_evergreen_reminders(db_session=None):
+    """Sends evergreen reminders for documents that have then enabled."""
+    documents = get_overdue_evergreen_documents(db_session=db_session)
+    log.debug(f"Documents that need reminders. NumDocuments: {len(documents)}")
+
+    if documents:
+        contact_fullname = contact_weblink = DISPATCH_HELP_EMAIL
```

Comment:
Any reason why we do the assignment here and pass them as parameters instead of using them directly in `create_reminder()`?